install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -2338,6 +2338,11 @@ envoy:
       maxUnavailable: 2
   # -- Roll out cilium envoy pods automatically when configmap is updated.
   rollOutPods: false
+  # -- Go off-road and bring your own custom Envoy bootstrap ConfigMap. Provide the name of the ConfigMap with a `bootstrap-config.json` key here.
+  # @schema
+  # type: [null, string]
+  # @schema
+  overrideBootstrapConfigMap: null
```

Comment:
Minor suggestion: name this `bootstrapConfigMap` and have it's default value be `~`.

Reply:
sounds great, ill push an update. Is the `~` a stylistic choice?